statue.gemspec

```diff
@@ -22,4 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rack", "~> 1.6"
   spec.add_development_dependency "rack-test", "~> 0.6.3"
   spec.add_development_dependency "minitest"
+  spec.add_development_dependency "sidekiq"
+  spec.add_development_dependency "pry-rescue"
```

Comment:
**pry-rescue** really helped me while fixing the specs, so I decided to leave it as a development dependency (open to discussion though).